tests/scalar_advection/scalar_advection_adaptive.py

```diff
@@ -0,0 +1,134 @@
+# Demo for pure scalar advection with adaptive timestepping - this is adapted from the Firedrake DG_advection demo,
+# but here we use G-ADOPT's GenericTransportSolver with IrksomeRadauIIA adaptive timestepping and use a CG discretisation with
```

Comment:
How about putting a full stop at the end of the first line and then starting the second line with `Here, however`?